noir-projects/noir-contracts/contracts/test/txe_test_contract/src/test.nr

```diff
@@ -1,50 +1,50 @@
 use crate::TXETest;
 
-use dep::aztec::test::helpers::test_environment::TestEnvironment;
-
-use aztec::macros::functions::initializer;
+use aztec::{
+    macros::functions::initializer,
+    note::{note_getter::{MAX_NOTES_PER_PAGE, view_notes}, note_viewer_options::NoteViewerOptions},
+    oracle::notes::set_sender_for_tags,
+    protocol_types::{address::AztecAddress, storage::map::derive_storage_slot_in_map},
+    test::helpers::test_environment::TestEnvironment,
+};
 use value_note::{balance_utils, value_note::ValueNote};
 
-use dep::aztec::note::note_getter::{MAX_NOTES_PER_PAGE, view_notes};
-use dep::aztec::note::note_viewer_options::NoteViewerOptions;
-use dep::aztec::protocol_types::{address::AztecAddress, storage::map::derive_storage_slot_in_map};
-
 pub unconstrained fn setup(
-    initial_value: Field,
-) -> (&mut TestEnvironment, AztecAddress, AztecAddress, AztecAddress) {
+    initial_value: u64,
```

Comment:
There was no reason to not have the `initial_value` be `u64` here so I changed it.